ArduPlane/Parameters.cpp

```diff
@@ -637,35 +637,34 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Advanced
     GSCALAR(scaling_speed,        "SCALING_SPEED",    SCALING_SPEED),
 
-    // @Param: MIN_GNDSPD_CM
+    // @Param: MIN_GROUNDSPEED
     // @DisplayName: Minimum ground speed
     // @Description: Minimum ground speed in cm/s when under airspeed control
-    // @Units: cm/s
+    // @Units: m/s
     // @User: Advanced
-    ASCALAR(min_gndspeed_cm,      "MIN_GNDSPD_CM",  MIN_GNDSPEED_CM),
+    ASCALAR(min_groundspeed,      "MIN_GROUNDSPEED",  MIN_GROUNDSPEED),
 
-    // @Param: TRIM_PITCH_CD
+    // @Param: PTCH_TRIM_DEG
```

Comment:
Just FYI, I don't think it matters too much if it is PITCH_TRIM_xxx or TRIM_PITCH_xxx (or some shortened form of either).  It's just a matter of whether you think of "TRIM" as a feature or not.  E.g. would users want to set all their trims at the same time?  If "yes" it might be considered a feature and "Trim" would go at the front.  Either way is fine with me of course.

Reply:
I think it should be grouped with other pitch control parameters